modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/styles/_dnd_table.scss

```diff
@@ -140,6 +140,8 @@
 
 	&.fixed {
 		display: block;
+		overflow: auto;
+		white-space: nowrap;
```

Comment:
Isn't `overflow: auto;` enough to fix the issue? I would expect white space wrapping to be the standard behavior. This may then be overriden for some specific fields in usages, fields that break on wrapping for whatever reason.

Reply:
Makes sense @markocikos! I removed the `white-space: nowrap;` from the code, Thanks for the review!